libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -290,6 +306,12 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
      * @hidden
      */
     onValueChange($event: string): void {
+        // when search result not changed but input text is changed.
+        // again need to announce the result, so clear this message.
+        setTimeout(() => {
```

Comment:
This could be simplified:

```suggestion
        setTimeout(() => this._currentSearchSuggestionAnnoucementMessage = '');
```

Reply:
fixed